### ISSUE TITLE
Getting diamonds although not present in config

### ISSUE THREAD

**Issue by Bloodlex** [Xenoyia/XenTradingCards#3]
I'm getting diamonds as a reward although there are no diamonds in any of the configs

**Comment by sarhatabaot** [comment]
Could you post your config here via pastebin?

**Comment by Bloodlex** [comment]
Yes of course, here you are: [https://pastebin.com/m5cNDdVc](https://pastebin.com/m5cNDdVc)

I think something is also wrong with rarity names, because I can get rewarded for rarity name that is also not in the config.

Thanks a lot!!

**Comment by sarhatabaot** [comment]
Looks like it's a problem with the way tradingcards accesses the config files. Try using my fork and see if it helps.
https://github.com/TreasureIslandMC/XenTradingCards
I'll publish a pre-release for you to easily download without building it yourself.

**Comment by Bloodlex** [comment]
> Looks like it's a problem with the way tradingcards accesses the config files. Try using my fork and see if it helps.
> https://github.com/TreasureIslandMC/XenTradingCards
> I'll publish a pre-release for you to easily download without building it yourself.

Thanks, but it gives me only shiny versions and doesn't save the deck properly (even if I forced it to give me only normal card versions and turning shiny off for that cards):

Decks:
  Inventories:
    b416b3e5-4d14-4299-9d75-dce8759f5bbb:
      '0':
      - nullEpicka,None,1,no
      - nullEpicka,None,1,no
      - nullEpicka,None,1,no
      - nullEpicka,None,1,no

It seems also that part of my config.yml has been cut out. I've also attached my cards.yml.
With cards rewards it says there is no such rarity.

cards.yml [https://pastebin.com/TS3wR2Na](https://pastebin.com/TS3wR2Na)
config.yml [https://pastebin.com/q0p5yHM3](https://pastebin.com/q0p5yHM3)

Could you please take a look?
